docs/intro/tutorial01.txt

```diff
@@ -48,15 +48,23 @@ including database configuration, Django-specific options and
 application-specific settings.
 
 From the command line, ``cd`` into a directory where you'd like to store your
-code, then run the following command:
+code and create a new directory named ``djangotutorial``.
 
 .. console::
 
-   $ django-admin startproject mysite
+   $ mkdir djangotutorial
 
-This will create a ``mysite`` directory in your current directory. If it didn't
+Then, run the following command to bootstrap a new Django project:
+
+.. console::
+
+   $ django-admin startproject mysite djangotutorial
+
+This will create your project in the ``djangotutorial`` directory.  Its
+name doesn't matter to Django; you can rename it to anything you like. If it didn't
```

Comment:
```suggestion
This will create a project called ``mysite`` in the ``djangotutorial`` directory. 
```

I would leave the `Its name doesn't matter to Django...` where it was.